api/src/main/java/org/openmrs/api/VisitService.java

```diff
@@ -258,6 +259,15 @@ public List<Visit> getVisits(Collection<VisitType> visitTypes, Collection<Patien
 	        Date minEndDatetime, Date maxEndDatetime, Map<VisitAttributeType, Object> attributeValues,
 	        boolean includeInactive, boolean includeVoided) throws APIException;
 	
+	/**
+	 * Gets the visits matching the specified search criteria
+	 * 
+ 	 * @param visitSearchCriteria
+	 * @return
+	 * @throws APIException
+	 */	
+	@Authorized(PrivilegeConstants.GET_VISITS)
+	public List<Visit> getVisits(VisitSearchCriteria visitSearchCriteria) throws APIException;
```

Comment:
nit: blank line between function definitions

```suggestion
	public List<Visit> getVisits(VisitSearchCriteria visitSearchCriteria) throws APIException;

```